benchto-driver/src/main/java/io/trino/benchto/driver/execution/QueryExecutionDriver.java

```diff
@@ -104,11 +101,22 @@ private QueryExecutionResult executeSelectQuery(
         }
     }
 
-    private QueryExecutionResult executeUpdateQuery(Connection connection, QueryExecutionResultBuilder queryExecutionResultBuilder, String sqlStatement)
+    private QueryExecutionResult executeUpdateQuery(
+            Connection connection,
+            QueryExecutionResultBuilder queryExecutionResultBuilder,
+            String sqlStatement,
+            Optional<Path> resultFile)
             throws SQLException
     {
         try (Statement statement = connection.createStatement()) {
+            if (statement.isWrapperFor(TrinoStatement.class)) {
```

Comment:
Do we use this code path for connecting to non-Trino databases as well ?

Reply:
yes, as I understand this it can be.
Same thing is done for select queries: https://github.com/trinodb/benchto/blob/master/benchto-driver/src/main/java/io/trino/benchto/driver/execution/QueryExecutionDriver.java#L90